src/components/Viewer/index.tsx

```diff
@@ -157,7 +163,8 @@ const RenderViewer: React.FC<CloverViewerProps> = ({
       if (manifests.length > 0) {
         dispatch({
           type: "updateActiveManifest",
-          manifestId: manifests[0],
+          manifestId:
+            decodeContentStateManifestURI(iiifContent) || manifests[0],
```

Comment:
Likewise with Canvas, Clover has issues of the Manifest doesn't exist on the Collection. I wonder if we can work this into a single helper: 

```suggestion
  getActiveManifest(iiifContent, manifests),
```

and then in `iiif.ts`

```ts
export const getActiveManifest = (iiifContent: string, manifests: string[]) => {
  const manifest = decodeContentStateManifestURI(iiifContent);
  return manifests.includes(manifest) ? manifest : manifests[0];
};
```